server/controllers/fast-food.js

```diff
@@ -0,0 +1,21 @@
+import { allFastFoodData } from '../models/fast-food-data';
+
+/**
+ * Represents a Fast Food.
+ */
+class FastFood {
+    /**
+     * This function gets all fastfood
+     * @param {object} req - the request file.
+     * @param {object} res - The response file.
+     * @returns {object} Returns the status and diary information.
+     */
+    allFastFood(req, res) {
```

Comment:
I am a little confused about what this method does. I assumed you were returning orders that have been made, but instead you are returning all the food you have. I think there is a difference between the food and the orders. Please clarify?